### ISSUE TITLE
When I console.log my results they are correct, however the challenge doesn't accept them.

### ISSUE THREAD

**Issue by smholsen** [FreeCodeCamp/FreeCodeCamp#7547]
Challenge [Where do I belong](http://www.freecodecamp.com/challenges/where-do-i-belong#?solution=function%20where(arr%2C%20num)%20%7B%0A%20%20%20%20%2F%2F%20Find%20my%20place%20in%20this%20sorted%20array.%0A%20%20%20%20arr.sort(function(a%2C%20b)%7Breturn%20a-b%3B%7D)%3B%0A%0A%20%20%20%20for%20(var%20listIndex%20in%20arr)%7B%0A%20%20%20%20%20%20%20%20if%20(num%20%3C%3D%20arr%5BlistIndex%5D)%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20return%20listIndex%3B%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%0A%20%20%20%20return%20arr.length%3B%0A%7D%0A%0Aconsole.log(where(%5B40%2C%2060%5D%2C%2050))%3B%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.116 Safari/537.36</code>.

![alt text](http://i.imgur.com/xQ3Djiy.png "1")
![alt text](http://i.imgur.com/YdrBjT4.png "2")

My code:
```javascript
function where(arr, num) {
    // Find my place in this sorted array.
    arr.sort(function(a, b){return a-b;});

    for (var listIndex in arr){
        if (num <= arr[listIndex]){
            return listIndex;
        }
    }

    return arr.length;
}

console.log(where([40, 60], 50));

```

**Comment by ltegman** [comment]
`for ... in` should not typically be used to iterate over arrays. This is one of the reasons. `for ... in` assigns the value of an objects key to the variable defined in the `for ... in` statement and all javascript object keys must be strings. Array syntax makes it seem like this isn't the case, but under the hood it's storing all the values of the array in a key-value pair with strings as keys.

In short, you're returning a string when you should be returning a number. Thanks and happy coding!

**Comment by smholsen** [comment]
Wow, I did not realize that. Thank you for the heads up :)
